Nice doc :)

ia3andy opened the pull request.

FroMage commented on docs/modules/ROOT/nav.adoc:
```diff
@@ -1 +1,23 @@
+include::./includes/attributes.adoc[]
+:config-file: application.properties
+
+* xref:index.adoc[Getting started]
```
Can't we generate this TOC? It's bound to be forgotten/out of sync, no?

ia3andy replied:
@gastaldi ? Could generate it from the `==` of all the pages?

gastaldi replied:
Maybe there is an Asciidoc macro or something that will generate the TOC based on the file. I know there is this: https://docs.asciidoctor.org/asciidoc/latest/toc/position/#at-macro but it doesn't seem to work for XREF links

FroMage reviewed (commented):
So, what exactly did you do to index? Just split it in two? The diffs don't show me what you did.

ia3andy commented:
> So, what exactly did you do to index? Just split it in two? The diffs don't show me what you did.

I:
- splitted
- added titles on each pages
- added manual fragments `[#...]`
- Created the Nav Manually

FroMage commented:
Why manual fragments instead of the auto-generated ones?

ia3andy commented:
> Why manual fragments instead of the auto-generated ones?

So they don't change if the title changes as they are referenced

FroMage commented:
OK, very good point :)

ia3andy commented:
@FroMage did you approve?

ia3andy commented:
(we can always switch to auto-generated if that become something)

ia3andy commented:
> > To test the nav locally I think you'd need to run Antora to generate the website by changing the [Antora playbook](https://github.com/quarkiverse/quarkiverse-docs/blob/main/antora-playbook.yml#L193) to point to your repository/branch
> 
> Actually it's worse than this. Now my generated `index.html` page contains nothing and I can't navigate and see the rest of the docs locally before I push anything.
> 
> How do I test my docs locally now?

I thought I broke something but yeah I think we need to have some nav generated locally


It seems to be nice:
https://docs.quarkiverse.io/quarkus-renarde/dev/index.html

FroMage commented:
Yeah, it _looks_ nice, on the surface, BUT… now I have two navs: one at the left which is missing deeper levels and is not generated, and one at the right which is missing higher levels but is generated.

This is really bad UI. How can we fix this?

ia3andy commented:
@gastaldi @FroMage 

I don't think it's BAD UI, people tend to see things differently. Still I think we should have a common experience, let's continue the discussion there where @holly-cummins has already started some comparison with other docs:
https://github.com/quarkiverse/quarkiverse-docs/issues/102#issuecomment-1640558021

ia3andy commented:
@gastaldi for some reason the config reference is not working, I think it wasn't already before my change any thoughts?

gastaldi commented:
@ia3andy I browsed the sources and didn't find any interface with `@ConfigMapping` or `@ConfigRoot`, what do you mean?

FroMage commented:
I seem to have also lost the favicon I recently added :(
